packages/shared/src/components/post/PostComments.tsx

```diff
@@ -51,14 +54,19 @@ export function PostComments({
   const container = useRef<HTMLDivElement>();
   const { tokenRefreshed } = useContext(AuthContext);
   const { requestMethod } = useRequestProtocol();
-  const queryKey = generateQueryKey(RequestKey.PostComments, null, id);
+  const queryKey = generateQueryKey(
+    RequestKey.PostComments,
+    null,
+    id,
+    String(sortBy),
```

Comment:
Why this cast?

Reply:
maybe alternative `{ sortBy }` so we can add other options in the future more easily without changing queryKey drastically.

Reply:
Just to be sure it is a string. Let me check if passing a bool directly should not be an issue.